_includes/sidebar.html

```diff
@@ -47,7 +48,7 @@
 
       <a class="sidebar-nav-item active" href="CppCoreGuidelines.html">C++ Core Guidelines</a>
       <a class="sidebar-nav-item active" href="index.html">README</a>
-      <a class="sidebar-nav-item active" href="CONTRIBUTING.html">Contributing</a>
+      <a class="sidebar-nav-item active" href="CONTRIBUTING.md">Contributing</a>
```

Comment:
I was actually looking at this earlier today and it wasn't clear to me whether this is the right fix, or maybe `CONTRIBUTING.md` needs to be added to `include` in `_config.yml`? It seems like the reason why `CppCoreGuidelines.html` is valid is because it's part of that list...?